lib/assessment.js

```diff
@@ -37,6 +75,12 @@ var any = function any(array, callback, thisArg) {
 // made up of the array elements for which invoking
 // the callback returns a truthy value
 var select = function select(array, callback, thisArg) {
+<<<<<<< HEAD
+var array = []
+array.map
+// No idea how to begin
+// .map() ????????
```

Comment:
Here, you are being asked to implement `Array.filter`. It takes a list and a callback, and returns a list with length up to the length of the passed-in list. The callback is used to determine which elements to keep and which to discard.

If I'm not helping someone else after 4:30, I'd be happy to guide you through writing this and clearing up what questions you have. If not, please ask Matt about this during 4:30-5:30, or take advantage of Tom's after-class sessions when he's holding them.